fluent-bundle/src/types/number.rs

```diff
@@ -8,6 +8,23 @@ use intl_pluralrules::operands::PluralOperands;
 use crate::args::FluentArgs;
 use crate::types::FluentValue;
 
+#[derive(Debug, Default, Copy, Clone, Hash, PartialEq, Eq)]
+pub enum FluentNumberType {
```

Comment:
Is there discussion somewhere I missed about adding this type parameter? I don't see this in the Fluent language spec for NUMBER, and it isn't part of `intl.NumberFormat` either. I understand why it is useful but I think we need to keep an eye on how this Rust implementation stacks up against the spec and other implementations. There might have been discussion or other implementations this is coming from I just don't know where that is/was.

Reply:
https://github.com/projectfluent/python-fluent/blob/bc5b0e52a059944503ff2624b8d0f9c285d81016/fluent.runtime/fluent/runtime/types.py#L32

https://github.com/projectfluent/fluent/blob/fd8f95478e29dda8121da7e275d375eb8dadbcb0/guide/selectors.md?plain=1#L42

Reply:
I also don't know where it comes from, but it's in the docs and in other implementations; thanks for the links, @waywardmonkeys.

Reply:
https://github.com/projectfluent/fluent/pull/259 is the pull that added this to the guide.

It was added to the Python implementation here: https://github.com/projectfluent/python-fluent/pull/193 with a note that it wasn't in the spec but was in the guide.